packages/opensrp-plans/src/containers/ListView/index.tsx

```diff
@@ -86,13 +86,11 @@ const PlansList = (props: PlansListTypes) => {
         <title>{pageTitle}</title>
       </Helmet>
       <PageHeader title={pageTitle} className="page-header"></PageHeader>
-      <Row className={'list-view'}>
+      <Row className={'list-view mission-view'}>
         <Col className={'main-content'}>
-          <div className="main-content__header">
+          <div className="main-content__header mission-header">
             <Link to={PLANS_CREATE_VIEW_URL}>
-              <Button type="primary" size="large">
-                {NEW_MISSION}
-              </Button>
+              <Button type="primary">{NEW_MISSION}</Button>
```

Comment:
Does changing the size make the font 14px?

Reply:
yes, default is set to 14px, So removing `size='large'` worked.

Reply:
looks like it, but Haseem please confirm

Reply:
<img width="358" alt="Screenshot 2021-02-25 at 10 38 53 PM" src="https://user-images.githubusercontent.com/68631902/109193514-42024700-77ba-11eb-8c5e-a6f3cb5b774f.png">